src/main/java/com/marek/onlinebookstore/service/order/OrderService.java

```diff
@@ -0,0 +1,20 @@
+package com.marek.onlinebookstore.service.order;
+
+import com.marek.onlinebookstore.dto.order.OrderDto;
+import com.marek.onlinebookstore.dto.order.OrderItemDto;
+import com.marek.onlinebookstore.dto.order.PlacingOrderRequestDto;
+import com.marek.onlinebookstore.model.User;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+
+public interface OrderService {
+    OrderDto makeOrder(PlacingOrderRequestDto requestDto);
```

Comment:
The method 'makeOrder' might require some form of user authentication or validation to ensure that only valid users can make orders.